spec/system/teams_page_system_spec.rb

```diff
@@ -7,6 +7,7 @@
 
 require 'rails_helper'
 
+require_relative '../../app/controllers/api/teams/members_controller'
```

Comment:
Why is this needed?

Reply:
When testing this spec alone, the members_controller is not defined, however it works when doing all specs. Autoload is a bit messy and this line makes sure, it's loaded. Will insert a comment.